src/internal/m365/collection/site/backup.go

```diff
@@ -154,7 +155,7 @@ func CollectLists(
 	var (
 		el   = errs.Local()
 		spcs = make([]data.BackupCollection, 0)
-		acc  = api.CallConfig{Select: idAnd("list")}
+		acc  = api.CallConfig{Select: idAnd("list", "lastModifiedDateTime")}
```

Comment:
nit: Can we call this `cfg` instead of `acc`? Easier to understand and used in other places too.

Reply:
made the change in #4945